File: api/Fitr/Models/Session.cs
using System;
using System.Collections.Generic;

namespace Fitr.Models
{
    public class Session
    {
        public int Id { get; set; }

        public DateTime Date { get; set; }
        
        public int WorkoutId { get; set; }
        
        public virtual Workout Workout { get; set; }

        public virtual ICollection<Set> Sets { get; set; }
    }
}

File: lib/models/session.dart
class Session {
  int id;
  DateTime date;
  int workoutId;

  Session({this.id, this.date, this.workoutId});

  factory Session.fromJson(Map<String, dynamic> json) {
    return Session(
      id: json['id'],
      date: json['date'],
      workoutId: json['workoutId'],
    );
  }
}
